File: src/main/java/com/uwetrottmann/trakt5/entities/Stats.java
package com.uwetrottmann.trakt5.entities;

public class Stats {

    public Integer watchers;
    public Integer plays;
    public Integer collectors;
    public Integer comments;
    public Integer lists;
    public Integer votes;

    /** Specific to shows, seasons and episodes. */
    public Integer collected_episodes;

}

File: src/main/kotlin/com/redissi/trakt/entities/Stats.kt
package com.redissi.trakt.entities

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
class Stats(
    val watchers: Int? = null,
    val plays: Int? = null,
    val collectors: Int? = null,
    val comments: Int? = null,
    val lists: Int? = null,
    val votes: Int? = null,
    /** Specific to shows, seasons and episodes.  */
    @Json(name = "collected_episodes")
    val collectedEpisodes: Int? = null
)